pkg/common/volume_operations.go

```diff
@@ -276,7 +280,9 @@ func (vo *VolumeOperationsImpl) handleVolumeInProgress(ctx context.Context, log
 			log.Errorf("Timeout of %s for volume creation exceeded.", base.DefaultTimeoutForVolumeOperations)
 			volumeCR.Spec.CSIStatus = apiV1.Failed
 			// todo don't ignore error here
-			_ = vo.k8sClient.UpdateCRWithAttempts(ctx, volumeCR, 5)
+			_ = vo.k8sClient.UpdateCR(ctx, volumeCR, &k8s.KubeClientRequestOptions{
+				MaxBackoffRetries: &vo.updateCRAttempts,
```

Comment:
Why not log this error instead of ignoring?

Reply:
logged this